cli/cli/kurtosis_config/overrides_objects/v5/grafloki_config_v5.go

```diff
@@ -0,0 +1,7 @@
+package v5
+
+type GraflokiConfig struct {
+	ShouldStartBeforeEngine bool   `yaml:"should-start-before-engine,omitempty"`
```

Comment:
If there's a good reason for this config to exist, maybe comments explaining why one would ever pick one or the other?

Reply:
will add a comment